.github/linter/Dangerfile

```diff
@@ -28,6 +28,19 @@ files.each do |f|
     end
 end
 
+files = git.added_files + git.modified_files
```

Comment:
Revert this file (this change is probably from master, but it's best to add it when merging rather than now, since it can cause merge conflicts)

Reply:
This was caused by the merge with branch with master, soo git should now the history and not to cause a merge conflict